### PR TITLE
Renamed leftover strings to Cuberite / Server, as needed.

### PR DESCRIPTION
**Pull request by madmaxoft**
Also upgraded the user setting file for MSVC to 2013.

### CODE REVIEW

**Comment by SphinxC0re** [comment]
You missed a few newlines on some files

**Comment by madmaxoft** [comment]
The newlines weren't there in the first place, so I didn't add them. The `Cuberite.vcxproj.user` file is even created by MSVC itself, so I don't think it'd be a good idea to edit it manually.

File: Server/Plugins/ChatLog [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 983d23ca37baa89f7e4dc11d71502d9c059f6376
+Subproject commit cc04da381b9aa683ee3bbd7013f9d3fd909db039
```
Comment (SphinxC0re) [inline Server/Plugins/ChatLog]: Is this change intentional?

Comment (madmaxoft) [inline Server/Plugins/ChatLog]: Yes, all submodule changes are intentional, there were substitutions in the plugins as well.
